gruntfile_cfg.js

```diff
@@ -0,0 +1,176 @@
+      flow: {
+        steps: {
+          js: ['concat'],
+          css: ['concat']
+        }
+      }
+    }
+  },
+  release: {
+    src: '<%= src.html %>',
+    options: {
+      root: '<%= distdir %>',
+      dest: '<%= distdir %>'
+    }
+  }
+};};
+
+exports.copy = () => { return {
+  bundle: {
+    files: [
+      {dest:'<%= distdir %>/js/',  src: ['app.js'],  expand: true, cwd: '.tmp/concat/js/' },
+      {dest:'<%= distdir %>/css/', src: ['app.css'], expand: true, cwd: '.tmp/concat/css/' }
+    ]
+  },
+  assets: {
+    files: [
+      {dest: '<%= distdir %>/fonts/',  src: '*.{ttf,woff,woff2,eof,svg}', expand: true, cwd: 'bower_components/bootstrap/fonts/'},
+      {dest: '<%= distdir %>/fonts/',  src: '*.{ttf,woff,woff2,eof,svg}', expand: true, cwd: 'bower_components/font-awesome/fonts/'},
+      {dest: '<%= distdir %>/fonts/',  src: '*.{ttf,woff,woff2,eof,svg}', expand: true, cwd: 'bower_components/rdash-ui/dist/fonts/'},
+      {dest: '<%= distdir %>/images/', src: '**',                         expand: true, cwd: 'assets/images/'},
+      {dest: '<%= distdir %>/ico',     src: '**',                         expand: true, cwd: 'assets/ico'}
+    ]
+  }
+};};
+
+exports.eslint = () => { return {
+  src: ['gruntfile.js', '<%= src.js %>'],
+  options: { configFile: '.eslintrc.yml' }
+};};
+
+exports.html2js = () => { return {
+  app: {
+    options: { base: '.' },
+    src: ['<%= src.tpl %>'],
+    dest: '<%= distdir %>/templates/app.js',
+    module: '<%= pkg.name %>.templates'
+  }
+};};
+
+exports.concat = () => { return {
+  vendor: {
+    files: {
+      '<%= distdir %>/css/<%= pkg.name %>.css': ['<%= src.cssVendor %>', '<%= src.css %>'],
+      '<%= distdir %>/js/vendor.js': ['<%= src.jsVendor %>'],
+      '<%= distdir %>/js/angular.js': ['<%= src.angularVendor %>']
+    }
+  },
+  dist: {
+    options: { process: true },
+    files: {
+      '<%= distdir %>/js/<%= pkg.name %>.js': ['<%= src.js %>', '<%= src.jsTpl %>'],
+      '<%= distdir %>/index.html': ['index.html']
+    }
+  }
+};};
+
+exports.uglify = () => { return {
+  dist: {
+    files: { '<%= distdir %>/js/<%= pkg.name %>.js': ['<%= src.js %>', '<%= src.jsTpl %>'] }
+  },
+  vendor: {
+    options: { preserveComments: 'some' }, // Preserve license comments
+    files: { '<%= distdir %>/js/vendor.js': ['<%= src.jsVendor %>'] ,
+             '<%= distdir %>/js/angular.js': ['<%= src.angularVendor %>']
+    }
+  }
+};};
+
+exports.postcss = () => { return {
+  build: {
+    options: {
+      processors: [
+        require('autoprefixer')({browsers: 'last 2 versions'}), // add vendor prefixes
+        require('cssnano')() // minify the result
+      ]
+    },
+    src: '<%= distdir %>/css/<%= pkg.name %>.css',
+    dest: '<%= distdir %>/css/app.css'
+  }
+};};
+
+exports.watch = () => { return {
+  build: {
+    files: ['<%= src.js %>', '<%= src.css %>', '<%= src.tpl %>', '<%= src.html %>'],
+    tasks: ['build']
+  }
+};};
+
+exports.replace = () => { return {
```

Comment:
We do not use ES6 in other parts of the codebase so I'd rather avoid mixing ES5/ES6. Please change it to ES5 declaration.